django_tenants/signals.py

```diff
@@ -50,3 +55,47 @@ def tenant_delete_callback(sender, instance, **kwargs):
 
     if instance.auto_drop_schema and schema_exists(instance.schema_name):
         instance._drop_schema(True)
+
+
+@receiver(post_save)
+def change_domain(sender, instance, **kwargs):
+    """Change domain in cache."""
+    if TENANT_CACHE_ENABLE is not True or not isinstance(
+        instance, get_tenant_domain_model()
+    ):
+        return
+
+    if TENANT_CACHE_ENABLE is True:
+        TENANTS_CACHE_DATA[instance.domain] = instance.tenant
+
+
+@receiver(post_delete)
+def delete_domain(sender, instance, **kwargs):
+    """Delete domain from cache."""
+    if TENANT_CACHE_ENABLE is not True or not isinstance(
+        instance, get_tenant_domain_model()
+    ):
+        return
+
+    try:
+        del TENANTS_CACHE_DATA[instance.domain]
+    except KeyError:
+        ...
+
+
+@receiver([post_delete, post_save])
+def delete_or_change_client(sender, instance, **kwargs):
+    """Delete client from cache."""
+    if TENANT_CACHE_ENABLE is not True or not isinstance(
+        instance, get_tenant_domain_model()
+    ):
+        return
+
+    # copy() is required
+    # (RuntimeError: dictionary changed size during iteration)
+    for hostname, tenant in TENANTS_CACHE_DATA.copy().items():
+        if tenant is None or tenant == instance:
+            try:
+                del TENANTS_CACHE_DATA[hostname]
+            except KeyError:
+                ...
```

Comment:
why not just use `pass` ? I find it a bit more clear and not everyone may be familiar with the `...` syntax.